src/couch_replicator_doc_processor.erl

```diff
@@ -154,8 +173,9 @@ start_link() ->
 
 
 init([]) ->
+    ok = config:listen_for_changes(?MODULE, nil),
```

Comment:
this was done to re-parse all documents and fix them up, if we don't do that we don't need this